packages/harmony/src/components/text/Text/Text.tsx

```diff
@@ -92,7 +93,10 @@ export const Text = forwardRef(
         lineHeight: typography.lineHeight[variantConfig.lineHeight[size]],
         // @ts-expect-error
         fontWeight: typography.weight[variantConfig.fontWeight[strength]],
-        ...('css' in variantConfig && variantConfig.css)
+        ...('css' in variantConfig && variantConfig.css),
+        ...(shadow && {
+          textShadow: '0px 1px 5px rgba(0, 0, 0, 0.50)'
+        })
```

Comment:
so there's only ever one shadow? nice

Reply:
Yeah for now it appears so. There is a chance we use the same one for icons, so in that case may want to incorporate it into the theme